build_scripts/windows/scripts/build.cmd

```diff
@@ -139,36 +139,8 @@ copy %REPO_ROOT%\build_scripts\windows\resources\CLI_LICENSE.rtf %BUILDING_DIR%
 copy %REPO_ROOT%\build_scripts\windows\resources\ThirdPartyNotices.txt %BUILDING_DIR%
 copy %REPO_ROOT%\NOTICE.txt %BUILDING_DIR%
 
-REM Remove .py and only deploy .pyc files
-pushd %BUILDING_DIR%\Lib\site-packages
-for /f %%f in ('dir /b /s *.pyc') do (
-    set PARENT_DIR=%%~df%%~pf..
-    echo !PARENT_DIR! | findstr /C:\Lib\site-packages\pip\ 1>nul
-    if !errorlevel! neq  0 (
-        REM Only take the file name without 'pyc' extension: e.g., (same below) __init__.cpython-310
-        set FILENAME=%%~nf
-        REM Truncate the '.cpython-310' postfix which is 12 chars long: __init__
-        REM https://stackoverflow.com/a/636391/2199657
-        set BASE_FILENAME=!FILENAME:~0,-12!
-        REM __init__.pyc
-        set pyc=!BASE_FILENAME!.pyc
-        REM Delete ..\__init__.py
-        del !PARENT_DIR!\!BASE_FILENAME!.py
-        REM Copy to ..\__init__.pyc
-        copy %%~f !PARENT_DIR!\!pyc! >nul
-        REM Delete __init__.pyc
-        del %%~f
-    ) ELSE (
-        echo --SKIP !PARENT_DIR! under pip
-    )
-)
-popd
-
-REM Remove __pycache__
-echo remove pycache
-for /d /r %BUILDING_DIR%\Lib\site-packages\pip %%d in (__pycache__) do (
-    if exist %%d rmdir /s /q "%%d"
-)
+REM replace .py with .pyc and remove __pycache__ dir to save space
+%BUILDING_DIR%\python.exe %REPO_ROOT%\scripts\use_pyc.py %BUILDING_DIR%\Lib\site-packages\ %PYTHON_VERSION%
```

Comment:
I use `use_pyc.py` to replace cmd scripts not only for readability, but also for bug fix.
`invoke==1.2.0` has a weird file: `invoke/completion/__pycache__/__init__.cpython-36.pyc`, and this script saves it as `__init_.pyc` in final build.

Reply:
Won't it be pretty hard to debug anything when there's no source code left?

Reply:
That's true, but I think most of the users never edit CLI source code for debug.
As evidence of this claim, we've removed the source code on Windows for years and no one complains about this.
But they do care about package size, we get lots of feedback in https://github.com/Azure/azure-cli/issues/7387.

For those who want to change source code, they need to install from pypi.